src/components/berths/Berth/BerthDetails/BerthDetails.scss

```diff
@@ -10,28 +10,18 @@ $berthIconSize: 42px;
     justify-content: flex-start;
     flex-wrap: wrap;
     height: 100%;
-    padding: 1em;
-    padding-top: 0px;
+    padding: 1em 0;
     white-space: nowrap;
-
-    @include for-tablet {
-      flex-direction: column;
-      padding: 1em 0;
-    }
   }
 
   display: flex;
-  flex-direction: row;
+  flex-direction: column;
```

Comment:
Are you sure that this works like before with these changes? I mean these are kinda big style/layout changes to the style, so it might break something.

Reply:
@rBoost: I can say this line have never been seen before, because it's override by `column` in most of the screen. And yes, i do tested them all in many screensize